template.go

```diff
@@ -32,6 +36,12 @@ func init() {
 			}
 			return fmt.Sprintf("ecschedule::<%s>", key)
 		},
+		"tfstate": func(key string) string {
+			return fmt.Sprintf("ecschedule::tfstate::<`%s`>", key)
+		},
+		"tfstatef": func(key string, args ...string) string {
+			return fmt.Sprintf("ecschedule::tfstatef::<`%s` `%s`>", key, strings.Join(args, "` `"))
```

Comment:
@Songmu 
Since we receive any arguments, we use split and join to maintain variable names...  
Do you have any other good ideas?